alert.go

```diff
@@ -45,16 +45,17 @@ const maxWeight = 1.2
 
 type AlertNode struct {
 	node
-	a           *pipeline.AlertNode
-	topic       string
-	anonTopic   string
-	handlers    []alert.Handler
-	levels      []stateful.Expression
-	scopePools  []stateful.ScopePool
-	states      map[models.GroupID]*alertState
-	idTmpl      *text.Template
-	messageTmpl *text.Template
-	detailsTmpl *html.Template
+	a            *pipeline.AlertNode
+	topic        string
+	anonTopic    string
+	handlers     []alert.Handler
+	levels       []stateful.Expression
+	scopePools   []stateful.ScopePool
+	states       map[models.GroupID]*alertState
+	idTmpl       *text.Template
+	messageTmpl  *text.Template
+	detailsTmpl  *html.Template
+	hostnameTmpl *text.Template
```

Comment:
Since this is specific to the Sensu handler the template logic should exists in services/sensu/service.go.

Have a look at the Alerta handler as it does something quite similar to this.